x-pack/solutions/security/plugins/security_solution/common/entity_analytics/entity_store/constants.ts

```diff
@@ -25,4 +25,5 @@ export const ENTITY_STORE_REQUIRED_ES_CLUSTER_PRIVILEGES = [
 export const ENTITY_STORE_SOURCE_REQUIRED_ES_INDEX_PRIVILEGES = ['read', 'view_index_metadata'];
 
 // The index pattern for the entity store has to support '.entities.v1.latest.noop' index
-export const ENTITY_STORE_INDEX_PATTERN = '.entities.v1.latest.*';
+export const ENTITY_STORE_INDEX_PATTERN = '.entities.*';
```

Comment:
Why is it needed?

Reply:
Exactly that. It's for the permission checks on Entity Store init.